### ISSUE TITLE
Room Info's "Private" Toggle is Incorrect on Load/Refresh

### ISSUE THREAD

**Issue by maxdwit** [RocketChat/Rocket.Chat#5663]
Your Rocket.Chat version: 0.49.3

Create a private channel as user in the web gui with the "Channel  (+)" symbol on the top right.
Open up the "(i)" icon on the top right in this channel.
Look at the "private" setting. It isn't on.
Flip the private switch.
As administrator look at the room list for this chat ( look in private channels and normal channels ).
When you find it, select it. 
On the right side several fields are visible. 
All textfields are empty. One status and read-only are filled.
Enter a topic. It vanishs after saving.
go back to the user, open up the channel (i) again. The set topic is visible.
Now again flip the switch private to on.
Go back to the administrative page rooms. Room is now a normal channel.
From now on, when you switch from room<->private it works and the fields in the admin page aren't empty.

tl;dr: New private channels don't display the correct info menu on the top right and within the administrative/private rooms page. Flipping some switches around makes it work.

**Comment by graywolf336** [comment]
Basically the information pane on a private group doesn't show the correct value for the "Private" toggle. Can be seen on the demo server in any private group you're in. The private toggle is not "true" when it should be due to the value it is checking is actually a string and not a boolean.

**Comment by axeloz** [comment]
+1 
<img width="1276" alt="rocket_chat_" src="https://cloud.githubusercontent.com/assets/1597611/22202239/89488446-e167-11e6-93d3-39d73bf635f3.png">

**Comment by karlprieb** [comment]
Fixed in #5852
